heroic-core/src/main/java/com/spotify/heroic/HeroicCore.java

```diff
@@ -260,21 +268,59 @@ public HeroicCoreInstance newInstance() throws Exception {
 
     @SuppressWarnings("unchecked")
     private void enableTracing(Map<String, Object> config) throws IOException {
-        final Object stackdriverConfig = config.get("stackdriver");
-        if (stackdriverConfig != null) {
-            final Map<String, String> configMap = (Map) stackdriverConfig;
-            final String projectId = configMap.get("project");
+        final Object lightstepConfig = config.get("lightstep");
+        if (lightstepConfig != null) {
+            final Map<String, String> configMap = (Map) lightstepConfig;
+            final String collectorHost = configMap.getOrDefault("collectorHost",
```

Comment:
you can check now to see if collectorHost and accessToken are null and throw an exception that the required parameters are missing.